README.md

```diff
@@ -11,13 +11,21 @@ Here is a mock up of the future design:
 ![Alt text](design/images/TuneUp_Mockup_03_SortedByExecutionOrder.jpg?raw=true "TuneUp")
 
 ## Building
-Recommended build environment
+### Recommended Build Environment
 - VisualStudio 2019
 - .Net Framework 4.7 Developer Pack
 - Dynamo repository cloned and built on the same level of TuneUp repository which means your Dynamo repo and TuneUp repo should exist under the same parent folder.
 
+### Result Binaries
+- Everytime 3rd party dev build TuneUp in `Debug` config, they should expect:
```

Comment:
I think this can just 
```
say after a debug build of Tuneup one can expect:
```

no need to mention third party devs, same for anyone building this.